Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -1458,6 +1540,49 @@ final class SignatureVerificationHTTPClientTests: BaseHTTPClientTests {
         expect(response?.value?.verificationResult) == .verified
     }
 
+    func testCachedResponseDoesNotUpdateRequestDateIfNewResponseIsNotVerified() throws {
```

Comment:
Should we have a test testing that it does update if verified? It might be tricky though...

Reply:
That's tested by `testResponseFromServerUpdatesRequestDate`.

Reply:
But I guess it doesn't hurt to have another one where it's also verified, so let me do that.

Reply:
Done.